docs/endpoints/user.md

```diff
@@ -12,33 +14,100 @@ Create a new user.
 ```
 
 #### Body
-The body is a json format of a user.
+The body is a json format of a user, with an embedded reference to the Account the user will be linked to.
+Optionally, the Account (and underlying Network and Instance) can be created in one request.
```

Comment:
We used to, but dropped it when introducing accounts and instances. I believe the concept of an admin was disregarded or forgotten in the process, and when I found it missing we decided it's not profitable to reintroduce it. Mostly because most prospect users we talked to so far don't need it anyway.


> Op 4 aug. 2015 om 07:35 heeft Rick Kuipers <notifications@github.com> het volgende geschreven:
> 
> In docs/endpoints/user.md:
> 
> > @@ -12,33 +14,100 @@ Create a new user.
> >  ```
> >  
> >  #### Body
> > -The body is a json format of a user.
> > +The body is a json format of a user, with an embedded reference to the Account the user will be linked to.
> > +Optionally, the Account (and underlying Network and Instance) can be created in one request.
> @rtuin I partially agree, I believe we have a concept of admins per network.
> 
> —
> Reply to this email directly or view it on GitHub.
>